include/GafferImage/ImageAlgo.inl

```diff
@@ -319,19 +319,13 @@ void parallelProcessTiles( const ImagePlug *imagePlug, const std::vector<std::st
 		const Gaffer::ThreadState &threadState = Gaffer::ThreadState::current();
 		{
 			ImagePlug::ChannelDataScope channelDataScope( threadState );
-			channelDataScope.setChannelName( channelNames[0] );
-			functor( imagePlug, channelNames[0], tileOrigin );
```

Comment:
We still have a comment on line 309 that talks about accessing one channel first on a single thread. We should remove that (the comment at 303-307 seems pretty accurate still though).